lib/java/ydoc-server/src/main/java/org/enso/ydoc/polyfill/web/Target_ExecutorsFactory.java

```diff
@@ -0,0 +1,37 @@
+package org.enso.ydoc.polyfill.web;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import java.util.function.Predicate;
+
+@TargetClass(
+    className = Target_ExecutorsFactory.CLASS_NAME,
+    onlyWith = Target_ExecutorsFactory.Available.class)
+final class Target_ExecutorsFactory {
```

Comment:
To use these substitutions one needs a JAR built out of https://github.com/helidon-io/helidon/pull/9271:
- download and `unzip` [helidon-webserver-4.1.0-SNAPSHOT.zip](https://github.com/user-attachments/files/17108146/helidon-webserver-4.1.0-SNAPSHOT.zip)
- copy the JAR to expected coursier location: `cp helidon-webserver-4.1.0-SNAPSHOT.jar ~/.cache/coursier//v1/https/repo1.maven.org/maven2/io/helidon/webserver/helidon-webserver/4.0.8/helidon-webserver-4.0.8.jar`

Then these substitutions activate.

Reply:
Substitutions removed while switching to Helidon 4.1.2 in ee82801